frontend/src/app/components/global-search/global-search.service.ts

```diff
@@ -40,7 +40,7 @@ export class GlobalSearchService {
   private _currentTab = new BehaviorSubject<any>('work_packages');
   public currentTab$ = this._currentTab.asObservable();
 
-  private _projectScope = new BehaviorSubject<any>('all');
```

Comment:
@wielinde This led to the first "to-do" I described. Due to this default, the title would claim that the search is in "all projects" if you wanted to search only in the current project and it's sub projects... I could not really figure out, why this happened only in this special case. Is it necessary to set this initialisation here?

Reply:
Thank you for that finding. Please rebase your branch as I have fixed that. 

I fixed another bug you reported (changing project scope from "this project" to "all")